src/main/resources/application.properties

```
@@ -13,3 +13,7 @@ spring.mail.username=${DISA_MAIL_USERNAME}
 task.cron=${DISA_NOTIFICATION_CRON}
 
 disa.notifier.rest.endpoint=${DISA_NOTIFIER_URL}
+
+disa.notifier.seafile.username=${USERNAME}
+disa.notifier.seafile.password=${PASSWORD}
```

Maybe use more descriptive names:

```suggestion
disa.notifier.seafile.username=${SEAFILE_USERNAME}
disa.notifier.seafile.password=${SEAFILE_PASSWORD}
```